### DISCUSSION TITLE
More Data in the island_islands table in MySQL

### DISCUSSION THREAD

**Discussion by kristofergeo** [BG-Software-LLC/SuperiorSkyblock2#2117] [Pending Feature Requests] [answered]
would it be possible to add 2 more columns to the `island_islands` table? One for the leader of the island that will return their actual name and not their uuid like it does now, and one that will return the level value of their island

Currently, that's how the table looks
![image](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/114158743/d3087816-250c-4448-a8d7-b2dbf5197683)

**Comment by Lemvs** [comment, marked as answer]
If you can check the leader's UUID, you can also check his name by searching for his UUID in the player table. Duplicating stored data does not make sense. The level is not stored in the database because it is calculated on an ongoing basis from the island blocks, and only they are stored in the database.
